web/src/compose_actions.ts

```diff
@@ -463,7 +463,6 @@ export let start = (raw_opts: ComposeActionsStartOpts): void => {
         compose_validate.warn_if_in_search_view();
     }
 
-    compose_recipient.check_posting_policy_for_compose_box();
```

Comment:
This is fine to remove as `compose_validate.validate_and_update_send_button_status` is called in `complete_starting_tasks`, which is a few lines below this.